tests/proposals.test.ts

```diff
@@ -0,0 +1,134 @@
+import { it, describe, expect } from "vitest";
+import { TEST_SERVER_URL } from "./global.setup";
+import { resetDatabase } from "../database";
+import { Proposal } from "../types/proposal";
+import { seedDatabase } from "./helpers/seedDatabase";
+import { TEST_USER } from "../helpers";
+import { beforeEach } from "node:test";
```

Comment:
I think vitest has a beforeEach function

Reply:
it does